internal/containerizedengine/types.go

```diff
@@ -67,6 +64,12 @@ var (
 			NoNewPrivileges: false,
 		},
 	}
+
+	// RuntimeMetadataName is the name of the runtime metadata file
+	RuntimeMetadataName = "distribution_based_engine"
+
+	// ReleaseNotePrefix is where to point users to for release notes
+	ReleaseNotePrefix = "https://docs.docker.com/releasenotes"
```

Comment:
This should be a `const`.  It's also not used outside this package, so should not be exported

Reply:
I wanted to use this elsewhere, but realized that's going to mess up the recent changes to abstract the dependency and didn't have time to wire it through.  I really think that refactoring was the wrong thing to do, but I'm trying not to break it if I can avoid it.

Reply:
found a better way to move this up without breaking the abstraction.